inc/dashboard/grid.class.php

```diff
@@ -1194,6 +1194,40 @@ public function getAllDasboardCards($force = false): array {
             }
          }
 
+         foreach (get_declared_classes() as $itemtype) {
```

Comment:
Some `Item_Devices` class may not be declared yet at this point, so I guess it should be better to rely on `$CFG_GLPI['itemdevices']`.